.github/workflows/pr-workflow.yml

```diff
@@ -123,11 +131,85 @@ jobs:
           aws cloudformation create-stack --stack-name $STACK_NAME --template-url $TEMPLATE --parameters ParameterKey=AdminEmail,ParameterValue=$EMAIL ParameterKey=DataBucketName,ParameterValue=$DATA_BUCKET_NAME ParameterKey=AmcEndpointUrl,ParameterValue=$AMC_API_ENDPOINT ParameterKey=DataUploadAccountId,ParameterValue=$DATA_UPLOAD_ACCOUNT_ID --capabilities CAPABILITY_IAM CAPABILITY_NAMED_IAM CAPABILITY_AUTO_EXPAND --disable-rollback
           aws cloudformation wait stack-create-complete --stack-name $STACK_NAME
 
+  integration_test:
+    needs:
+      - build
+      - deploy
+    runs-on: ubuntu-latest
+    outputs:
+      AMC_GLUE_JOB_ROLE_NAME: ${{ steps.api_integration_test.outputs.AMC_GLUE_JOB_ROLE_NAME }}
+    steps:
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+      - name: Initialize AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{needs.build.outputs.REGION}}
+      - name: Check out pr branch
+        uses: actions/checkout@v2.3.4
+        with:
+          ref: ${{ github.sha }}
+      - name: api_integration_test
+        run: |
+          STACK_NAME=${{needs.deploy.outputs.STACK_NAME}}
+          SOLUTION_NAME=${{needs.build.outputs.SOLUTION_NAME}}
+          VERSION=${{needs.build.outputs.VERSION}}
+          RESOURCE_ID=$(aws cloudformation describe-stack-resources --stack-name $STACK_NAME --logical-resource-id GlueStack --query "StackResources[0].PhysicalResourceId")
+          echo $RESOURCE_ID
```

Comment:
Every time you echo something, you need to also echo the name of what you are echoing, otherwise it will be hard to decipher the log.

Reply:
ok i will fix that